tools/ldb_cmd.cc

```diff
@@ -1284,10 +1284,10 @@ void InternalDumpCommand::DoCommand() {
     if (max_keys_ > 0 && count >= max_keys_) break;
   }
   if(count_delim_) {
-    fprintf(stdout,"%s => count:%lld\tsize:%lld\n", rtype2.c_str(),
-        (long long)c,(long long)s2);
+    fprintf(stdout,"%s => count:%" PRIu64 "\tsize:%" PRIu64 "\n", rtype2.c_str(),
+        c,s2);
   } else
-  fprintf(stdout, "Internal keys in range: %lld\n", (long long) count);
+  fprintf(stdout, "Internal keys in range: %" PRIu64 "\n", (uint64_t)count);
```

Comment:
If count is not uint64_t, you don't have to cast it, use the right format to its type. By the way, C-style casting is not encourage in Google C++ style we are following. See https://google.github.io/styleguide/cppguide.html#Casting